lib/transcoder.ex

```
@@ -50,7 +50,12 @@ defmodule Membrane.Transcoder do
   end
 
   @impl true
-  def handle_child_notification({:stream_format, format}, :forwarding_filter, _ctx, state) do
+  def handle_child_notification(
+        {:stream_format, format},
+        :forwarding_filter,
+        _ctx,
+        %{input_stream_format: nil} = state
+      ) do
```

Let's raise a meaningful error saying that Transcoder doesn't support receiving stream format more than once